### PR TITLE
Assignment 7 - Bryan Caesar

### PR DESCRIPTION
**Pull request by bcaesar42**

### CODE REVIEW

File: SecretSanta/src/SecretSanta.Web/Controllers/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    catch(SwaggerException se)
+                    {
+                        ViewBag.ErrorMessage = se.Message;
+                    }
+                }
+            }
+
+            return result;
+        }
+    }
```
Comment (tdang2018) [inline SecretSanta/src/SecretSanta.Web/Controllers/UsersController.cs]: Hi Bryan. I did not see 2 functions about Update and Delete user in this file.

File: SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                Secret Santa
+            </a>
+        </div>
+
+        <div id="navbarBasicExample" class="navbar-menu">
+            <div class="navbar-start">
+                <a class="navbar-item" asp-controller="Users">
+                    Users
+                </a>
+                <a class="navbar-item">
```
Comment (tdang2018) [inline SecretSanta/src/SecretSanta.Web/Views/Shared/_Layout_Template.cshtml]: I think you should add asp-controller="Groups" inside tag <a> for Groups

File: SecretSanta/src/SecretSanta.Web/Controllers/UsersController.cs (hunk 1 above)
Comment (breaman) [reply SecretSanta/src/SecretSanta.Web/Controllers/UsersController.cs]: I would agree here, I don't see the code that allows me to edit a user's information or delete them either.

**Comment by breaman** [comment]
Not sure exactly what happened, but looks like there is no Groups controller that should have allowed me to create/edit/view groups. Did you have git issues again with this assignment?

**Pull request closed without merging**